validator/validator-main.protoascii

```diff
@@ -3303,6 +3303,14 @@ attr_lists: {
   }
 }
 
+# Video Interface attributes. These attributes can be attached to any
+# element that implements the video interface
+attr_lists: {
+  name: "implements-video-interface"
```

Comment:
@aghassemi @wassgha Minor nit: It may be worth renaming this to be more explicit if this is only for AMP and not AMP4ADS as video-interface sounds very generic. At least update the comment stating it's for `html_format: AMP` only.